<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWindow</class>
 <widget class="QWidget" name="AboutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>About Grabber</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/resources.qrc">:/images/icon.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string notr="true">&lt;span style=&quot; font-size:30pt; font-weight:600; color:#505050;&quot;&gt;Grabber&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCurrent">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCreator">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Grabber is a Bionus' creation.&lt;br/&gt;Please visit the &lt;a href=&quot;{website}&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;site&lt;/span&gt;&lt;/a&gt; to stay updated, or retrieve site or translations files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Special thanks to YMI for his help looking and solving bugs, as well as suggesting new features for the program.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCredits">
       <property name="text">
        <string>Russian translation byr Николай Тихонов.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
